KsApi/ServiceType.swift

```diff
@@ -77,6 +77,9 @@ public protocol ServiceType {
   func createBacking(input: CreateBackingInput) ->
     SignalProducer<CreateBackingEnvelope, ErrorEnvelope>
 
+  func completeOnSessionCheckout(input: GraphAPI.CompleteOnSessionCheckoutInput) ->
+    SignalProducer<GraphAPI.CompleteOnSessionCheckoutMutation.Data, ErrorEnvelope>
```

Comment:
I'd like to move away from wrapper objects unless we really need them - implementing this as its raw GraphQL types for now; if we need to write a wrapper later, I'm OK doing that just-in-time.